### ISSUE TITLE
Change XPath to take a list of regular expressions for a given key

### ISSUE THREAD

**Issue by GuiForget** [DigitalPebble/storm-crawler#123]
Results would be a flatmap of the individual results

**Comment by GuiForget** [comment]
@jnioche does that make sense? I was thinking of working on that over the week-end

**Comment by jnioche** [comment]
Do you mean have more than one xpath expression per label e.g. description below?

```
params": {
        "canonical": "//*[@rel=\"canonical\"]/@href",
        "description": "//*[@name=\"description\"]/@content"
      }
```

**Comment by GuiForget** [comment]
Yeah. So I would change it to this:
```
params": {
        "canonical": "//*[@rel=\"canonical\"]/@href",
        "description":  ["//*[@name=\"description\"]/@content", "//*[@name=\"other-description\"]/@content"]
 }
```

Basically single single string and list of string would be supported for backward compatibility

**Comment by jnioche** [comment]
makes a lot of sense.

**Comment by jnioche** [comment]
Implemented in #153
